django/forms/renderers.py

```diff
@@ -58,7 +52,10 @@ class Jinja2(EngineMixin, BaseRenderer):
     Load Jinja2 templates from the built-in widget templates in
     django/forms/jinja2 and from apps' 'jinja2' directory.
     """
-    backend = Jinja2
+    @property
```

Comment:
Could use `cached_property` to avoid the implicit `sys.modules` lookup on every access.

Reply:
Ah yes. Using `@cached_property` now.